int64 totalPhysMem
int64 physMemFree
int64 physMemUsed
int64 totalVirtualMem
int64 virtualMemFree
int64 virtualMemUsed
float32 cpuUsage
